composer.json

```diff
@@ -31,6 +31,7 @@
         "symfony/console": "^2.8 || ^3.2 || ^4.0",
         "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
         "symfony/event-dispatcher": "^2.8 || ^3.2 || ^4.0",
```

Comment:
With these changes, there are no usages left from `Symfony\Component\EventDispatcher\` namespace.
Should not we remove the dependency too?

Reply:
Not sure about this. We would remove an implementation for this thing: /

Reply:
I think people using it should require it. IMO we should remove it.